gnuradio-runtime/CMakeLists.txt

```diff
@@ -124,14 +123,14 @@ install(
     COMPONENT "runtime_runtime"
 )
 
-if(ENABLE_GR_LOG AND HAVE_LOG4CPP)
+if(ENABLE_GR_LOG AND LOG4CPP_FOUND)
 install(FILES
```

Comment:
It wasn't indented before but it's a short section so I'll include indenting it in the next push I do for this PR. It's not quite "there" yet if you followed the discussion. I'm almost there with overloading CMake's default `include_directories` ... very cool to be able to make a 'smart' version.